cst.yml

```
@@ -93,3 +95,7 @@ fileExistenceTests:
   - name: "Helm Secrets plugin"
     path: '/home/jenkins/.local/share/helm/plugins/helm-secrets'
     shouldExist: true
+  - name: 'doctl'
+    path: '/usr/bin/doctl'
```

```suggestion
    path: '/usr/local/bin/doctl'
```
 As per https://github.com/jenkins-infra/docker-helmfile/pull/63/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R65